lib/Cleantalk/Custom/Antispam/Cleantalk.php

```diff
@@ -0,0 +1,859 @@
+<?php
```

Comment:
Why this class is in `Custom` namespace? Is not it really enough the common class?